examples/calibration/train_mushroom_edibility_calibrated.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python
+
+import copy
+import logging
+import shutil
+
+import numpy as np
+import yaml
+
+import ludwig.visualize
+from ludwig.api import LudwigModel
+from ludwig.datasets import mushroom_edibility
+
+# clean out prior results
+shutil.rmtree("./results_mushroom_edibility", ignore_errors=True)
+shutil.rmtree("./visualizations_mushroom_edibility", ignore_errors=True)
+
+# Download and prepare the dataset
+dataset = mushroom_edibility.load()
+
+# This dataset has no split, so add a split column
+dataset.split = np.random.choice(3, len(dataset), p=(0.7, 0.1, 0.2))
+
+config_yaml = """
+output_features:
```

Comment:
Nit: should we define output_features after input_features for constistency/readability sake?